lib/oli/rendering/content/html.ex

```diff
@@ -155,6 +157,42 @@ defmodule Oli.Rendering.Content.Html do
     ["<li>", next.(), "</li>\n"]
   end
 
+  def formula_class(false), do: "formula"
+  def formula_class(true), do: "formula-inline"
+
+  def formula(context, next, properties, inline \\ false)
+
+  def formula(
+        %Oli.Rendering.Context{} = _context,
+        nil,
+        %{"subtype" => "latex", "src" => src},
+        inline
+      ) do
+    ["<span class=\"#{formula_class(inline)}\">\\[", escape_xml!(src), "\\]</span>\n"]
+  end
+
+  def formula(
+        %Oli.Rendering.Context{} = _context,
+        nil,
+        %{"subtype" => "mathml", "src" => src},
+        inline
+      ) do
+    [
+      "<span class=\"#{formula_class(inline)}\">",
+      Scrubber.scrub(src, MathMLSanitizer),
+      "</span>\n"
+    ]
+  end
+
+  def formula(%Oli.Rendering.Context{} = _context, next, _, inline) do
+    # The catch-all formula will handle anything with a children property, which should always be richtext
+    ["<span class=\"#{formula_class(inline)}\">", next.(), "</span>\n"]
+  end
+
+  def formula_inline(context, next, map) do
+    formula(context, next, map, true)
```

Comment:
Delegating to the `formula` impl for inline Latex will end up rendering using the incorrect MathJax escape sequence.   I believe our MathJax is configured to use \\[ and \\] to delimit block level formula and \\( and \\) for inline formula.

Reply:
Good catch, new commit up.

![image](https://user-images.githubusercontent.com/333265/173595643-85fcb054-f0db-4b38-8433-40895e02398f.png)